api/client/commands.go

```diff
@@ -712,6 +712,16 @@ func (cli *DockerCli) CmdStart(args ...string) error {
 	utils.ParseFlags(cmd, args, true)
 
 	hijacked := make(chan io.Closer)
+	// Block the return until the chan gets closed
+	defer func() {
+		log.Debugf("End of CmdStar(), Waiting for hijack to finish.")
```

Comment:
typo: CmdStar**t**(); and if you are editing, maybe this could be "CmdStart() returned, defer waiting for hijack to finish" ?